src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -54,14 +58,14 @@ const LaunchAnalysisModal = ({
       const { submissionId } = await launch({
         isSnapshot: type === processSnapshotTable,
         workspace, config, selectedEntityType, selectedEntityNames, newSetName, useCallCache, deleteIntermediateOutputFiles, useReferenceDisks,
-        memoryRetryMultiplier: retryWithMoreMemory ? retryMemoryFactor : undefined,
+        memoryRetryMultiplier: retryWithMoreMemory ? retryMemoryFactor : undefined, userComment: _.trim(userComment),
         onProgress: stage => {
           setMessage({ createSet: 'Creating set...', launch: 'Launching analysis...', checkBucketAccess: 'Checking bucket access...' }[stage])
         }
       })
       onSuccess(submissionId)
     } catch (error) {
-      setLaunchError(await (error instanceof Response ? error.text() : error.message))
+      setLaunchError(await (error instanceof Response ? error.json().then(data => data.message) : error.message))
```

Comment:
This would show a pretty ugly error before, so pulling out just the message portion. Before:

![image](https://user-images.githubusercontent.com/484484/131392018-84fcb45c-0501-459c-8a94-d741b1f7d11b.png)